defaults/main.yml

```diff
@@ -59,7 +59,7 @@ logstash_major_ver: '5.x'  # Define major version to install
 # Defines the miniumum amount of memory (in MB) required to effectively run Logstash
 logstash_min_memory_required: 1024
 
-logstash_minor_ver: '1:5.2.0-1'  # Define minor version to install
```

Comment:
This was added this way in order to control the exact version to install. I have not seen an issue with this yet as I know of. Did you experience an issue? If so how can I duplicate the issue? Any info would be much appreciated.